### ISSUE TITLE
Feature Request: < lager scrollbar>

### ISSUE THREAD

**Issue by Reaper10** [Sn8z/Poddr#1540]
Can you make the scrollbar lager? I can't see vary well
<img width="441" alt="Screenshot 2023-09-15 184618" src="https://github.com/Sn8z/Poddr/assets/8137561/7a929b06-e894-4a6e-8da0-d5d464cad524">

**Comment by toby11** [comment]
yes I cannot see the scrollbar at all on Window 10 - its like there isnt one?

**Comment by toby11** [comment]
yes please make the scroll bar bigger - its tiny - I do not know what the obsession is with small/transparent/difficult to click scroll bars.
